pkg/sysctl/build.yml

```diff
@@ -1,6 +1,5 @@
 image: sysctl
 config:
   pid: "host"
-  readonly: true
```

Comment:
Why did you remove this one?

Reply:
@w9n see the last line of each bullet above. I got errors about adding confs in via the image scoped YAML when read-only

Reply:
Perhaps we need an issue for changing the filesystem to readonly only after the image file adds from the YAML?

Reply:
You can `mount --bind` files on a read only filesystem, this should work in your case aswell.

Reply:
OK. I'll get more information and report back.